src/truncate.jl

```diff
@@ -59,12 +62,7 @@ end
 
 ### Constructors
 function Truncated(d::UnivariateDistribution, l::T, u::T) where {T <: Real}
-    l < u || error("lower bound should be less than upper bound.")
-    T2 = promote_type(T, eltype(d))
-    lcdf = isinf(l) ? zero(T2) : T2(cdf(d, l))
-    ucdf = isinf(u) ? one(T2) : T2(cdf(d, u))
-    tp = ucdf - lcdf
-    Truncated(d, promote(l, u, lcdf, ucdf, tp, log(tp))...)
+    truncated(d, l, u)
 end
 
 Truncated(d::UnivariateDistribution, l::Integer, u::Integer) = Truncated(d, float(l), float(u))
```

Comment:
These should be removed completely - `Truncated` is deprecated below.

Reply:
I tried to remove `Truncated` but it causes a bunch of tests to fail (and I guess potentially some code from downstream users), so I left it here.